apps/back/composer.json

```diff
@@ -63,5 +65,8 @@
             "allow-contrib": false,
             "require": "6.3.*"
         }
+    },
+    "require-dev": {
+        "symfony/maker-bundle": "^1.50"
```

Comment:
![NO](https://user-images.githubusercontent.com/1766827/125260303-807b4700-e300-11eb-9d3e-e3d549d171bb.gif)

Reply:
C'est noté j'ai supprimé le package dans la review de la PR précédente